File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s)
{
    bool answer = true;
    if (s.length() == 4 || s.length() == 6)
    {
        for (int i = 0; i < s.length(); i++)
        {
            if (!(s[i] >= 48 && s[i] <= 57))
                answer = false;
        }
    }
    else
        answer = false;
    return answer;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        for (int i = 0; i < s.length(); i++) {
            if (!Character.isDigit(s.charAt(i))) return false;
        }
        
        return s.length() == 4 || s.length() == 6;
    }
}